src/appleseed/foundation/core/appleseed.cpp

```diff
@@ -30,6 +30,9 @@
 // Interface header.
 #include "appleseed.h"
 
+// appleseed.renderer headers.
```

Comment:
Including appleseed.renderer headers from appleseed.foundation is not allowed. See my remark above about the location of the versioning files.